### PR TITLE
feat: contribution page

### LINKED ISSUE #23: Contribute page
Implement the contribute page:

https://www.fastify.io/contribute/

image

## Challanges

- [ ] Integrate a plugin to get the `Good first issues`

## Tips

What would I do?
I would look for an existing Docusaurus plugin or I would build it (instead of integrating custom logic here)

Note that the actual code from the old website should be easily migrated: https://github.com/fastify/website/blob/master/src/website/content/js/contribute.js

### PR DESCRIPTION
**Pull request by Eomm**
closes #23 

When I change the layout dark/light I get this error:

```
storageUtils.ts?10b8:23 Uncaught RangeError: Maximum call stack size exceeded.
    at dispatchChangeEvent (storageUtils.ts?10b8:23:1)
    at Object.set (storageUtils.ts?10b8:131:1)
    at storeColorMode (colorMode.tsx?7cd1:47:1)
    at eval (colorMode.tsx?7cd1:72:1)
```

It is present even after the `build&serve` process. Any ideas would be appreciated

## Layout

image

image

### Error state

image

### Loading state

image

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by AugustinMauroy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,9 +25,9 @@
     "lint:style:fix": "npm run lint:style -- --fix"
   },
   "dependencies": {
-    "@docusaurus/core": "2.2.0",
-    "@docusaurus/plugin-client-redirects": "^2.2.0",
-    "@docusaurus/preset-classic": "2.2.0",
+    "@docusaurus/core": "^2.3.1",
```
Comment (Eomm) [inline package.json:28]: Upgrading the dep solved the following error 🎉 

```
storageUtils.ts?10b8:23 Uncaught RangeError: Maximum call stack size exceeded.
```

File: src/pages/contribute.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import HeroBanner from '@site/src/components/HeroBanner'
+
+import GoodFirstIssue from '@site/src/components/GoodFirstIssue'
+
+<HeroBanner title="Join the Fastify community" />
+
+Fastify is a growing and friendly community, and would be lucky to have your contributions! 🙇‍♂️  
+Contributions are always welcome, we only ask you follow the [Contribution Guidelines](https://github.com/fastify/fastify/blob/master/CONTRIBUTING.md) and the [Code Of Conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md).
```
Comment (HassanBahati) [inline src/pages/contribute.mdx:8]:
```suggestion
Contributions are always welcome, we only ask you to follow the [Contribution Guidelines](https://github.com/fastify/fastify/blob/master/CONTRIBUTING.md) and the [Code Of Conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md).
```

File: src/pages/contribute.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+If you don't know where to start you can have a look at the list of **good first issues** below.
```
Comment (HassanBahati) [inline src/pages/contribute.mdx:10]:
```suggestion
If you don't know where to start, you can have a look at the list of **good first issues** below.
```

**Review by HassanBahati — COMMENTED** [review]
looks great :clap: @Eomm

**Pull request merged**